pkg/apis/hive/v1/dnszone_types.go

```diff
@@ -137,9 +134,9 @@ type GCPDNSZoneStatus struct {
 
 // AzureDNSZoneStatus contains status information to specific to Azure Cloud DNS Zones
 type AzureDNSZoneStatus struct {
-	// ZoneName is the name of the zone in Azure Cloud DNS
+	// ZoneID is the ID of the zone in Azure Cloud DNS
 	// +optional
-	ZoneName *string `json:"zoneName,omitempty"`
+	ZoneID *string `json:"zoneName,omitempty"`
```

Comment:
I still do not see anywhere where this field is used.

Reply:
I think I will remove it entirely as I don't see a real benefit of using that field

Reply:
Removed